<launch>
    <node pkg="ibeo_scala" type="scala_node" name="scala_node" output="screen" >
        <param name="scala_type" type="int" value="2"/>
        <param name="device_destination_ip" type="string" value="192.168.0.204"/><!--for scala B3 using multicast-->
        <param name="device_ip" type="string" value="192.168.0.204"/><!--for scala B2-->
        <param name="port" type="string" value="12004"/>
        <param name="multicast_interface_ip" type="string" value="224.0.0.1"/> <!--all-hosts group-->
        <param name="use_multicast" type="bool" value="true"/>
        <param name="min_distance" type="double" value="1.0"/>
        <param name="log_level" type="string" value="Quiet"/>
        <!--log level
            "Debug, Debug2, Quiet, Error, Warning, Info, InfoDetails"-->
    </node>
</launch>
